pdc_client/plugins/auth_group_resource_permissions.py

```diff
@@ -0,0 +1,101 @@
+        self.add_create_update_argument(create_parser, required=True)
+        create_parser.set_defaults(func=self.group_resource_permission_create)
+
+        delete_parser = self.add_action('delete', help='delete an existing group resource permission')
+        delete_parser.add_argument('id', metavar='ID')
+        delete_parser.set_defaults(func=self.group_resource_permission_delete)
+
+    def add_create_update_argument(self, parser, required=True):
+        required_args = {'resource': {}, 'permission': {}, 'group': {}}
+        add_create_update_args(parser, required_args, {}, required)
+
+    def print_group_resource_permissions(self, args):
+        fmt = '{0:<5} {1:20} {2:90} {3}'
+        start_line = True
+        for obj in args:
+            if start_line:
+                start_line = False
+                print(fmt.format('ID', 'Group', 'Resource', 'Permission'))
+                print()
+            print(fmt.format(obj['id'], obj['group'], obj['resource'], obj['permission']))
+
+    def list_group_resource_permission(self, args):
+        filters = extract_arguments(args)
+        results = self.client.get_paged(self.client['auth/group-resource-permissions']._, **filters)
+        if args.json:
+            print(json.dumps(list(results)))
+            return
+        if results:
+            self.print_group_resource_permissions(results)
+
+    def group_resource_permission_info(self, args, id=None):
+        obj_id = id or args.id
+        results = self.client['auth/group-resource-permissions'][obj_id]._()
+        if args.json:
+            print(json.dumps(results))
+            return
+        fmt = '{0:20} {1}'
+        print(fmt.format('ID', results['id']))
+        print(fmt.format('Resource', results['resource']))
+        print(fmt.format('Permission', results['permission']))
+        print(fmt.format('Group', results['group']))
+
+    def group_resource_permission_update(self, args):
+        data = extract_arguments(args)
+        if data:
+            self.logger.debug('Updating group resource permission %s with data %r', args.id, data)
+            self.client['auth/group-resource-permissions'][args.id]._ += data
+        else:
+            self.logger.debug('Empty data, skipping request')
+        self.group_resource_permission_info(args)
+
+    def group_resource_permission_create(self, args):
+        data = extract_arguments(args)
+        self.logger.debug('Creating group resource permission with data %r', data)
+        response = self.client['auth/group-resource-permissions']._(data)
+        self.group_resource_permission_info(args, response['id'])
+
+    def group_resource_permission_delete(self, args):
+        data = extract_arguments(args)
+        self.logger.debug('Deleting : group resource permission %s', args.id)
+        self.client['auth/group-resource-permissions'][args.id]._("DELETE", data)
+
+
+PLUGIN_CLASSES = [AuthGroupResourcePermissionPlugin]
```

Comment:
Should class name and here be AuthGroupResourcePermissionsPlugin is better?  Because the file name is auth_group_resource_permissions.py. Make it consistent may looks better.